schedule/schedule-appengine/src/main/java/com/jasify/schedule/appengine/model/activity/Activity.java

```diff
@@ -114,6 +116,12 @@ public void setCurrency(String currency) {
         return activityTypeRef;
     }
 
+    public void setRepeatDetailsRef(RepeatDetails repeatDetails) { repeatDetailsRef.setModel(repeatDetails); }
```

Comment:
I usually don't add this method.  I keep only the getter.  Maybe your approach is better...